// @module-federation/utilities/src/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    var desc = Object.getOwnPropertyDescriptor(m, k);
    if (!desc || ("get" in desc ? !m.__esModule : desc.writable || desc.configurable)) {
      desc = { enumerable: true, get: function() { return m[k]; } };
    }
    Object.defineProperty(o, k2, desc);
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __exportStar = (this && this.__exportStar) || function(m, exports) {
    for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(exports, p)) __createBinding(exports, m, p);
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.importDelegatedModule = exports.getRuntimeRemotes = exports.Logger = exports.correctImportPath = exports.importRemote = exports.isObjectEmpty = exports.getModule = exports.injectScript = exports.getContainer = exports.createRuntimeVariables = exports.createDelegatedModule = void 0;
__exportStar(require("./types"), exports);
var common_1 = require("./utils/common");
Object.defineProperty(exports, "createDelegatedModule", { enumerable: true, get: function () { return common_1.createDelegatedModule; } });
Object.defineProperty(exports, "createRuntimeVariables", { enumerable: true, get: function () { return common_1.createRuntimeVariables; } });
Object.defineProperty(exports, "getContainer", { enumerable: true, get: function () { return common_1.getContainer; } });
Object.defineProperty(exports, "injectScript", { enumerable: true, get: function () { return common_1.injectScript; } });
Object.defineProperty(exports, "getModule", { enumerable: true, get: function () { return common_1.getModule; } });
var isEmpty_1 = require("./utils/isEmpty");
Object.defineProperty(exports, "isObjectEmpty", { enumerable: true, get: function () { return isEmpty_1.isObjectEmpty; } });
var importRemote_1 = require("./utils/importRemote");
Object.defineProperty(exports, "importRemote", { enumerable: true, get: function () { return importRemote_1.importRemote; } });
var correctImportPath_1 = require("./utils/correctImportPath");
Object.defineProperty(exports, "correctImportPath", { enumerable: true, get: function () { return correctImportPath_1.correctImportPath; } });
var Logger_1 = require("./Logger");
Object.defineProperty(exports, "Logger", { enumerable: true, get: function () { return Logger_1.Logger; } });
var getRuntimeRemotes_1 = require("./utils/getRuntimeRemotes");
Object.defineProperty(exports, "getRuntimeRemotes", { enumerable: true, get: function () { return getRuntimeRemotes_1.getRuntimeRemotes; } });
var importDelegatedModule_1 = require("./utils/importDelegatedModule");
Object.defineProperty(exports, "importDelegatedModule", { enumerable: true, get: function () { return importDelegatedModule_1.importDelegatedModule; } });
